### DISCUSSION TITLE
VK Play Live streamers waiting BetterTV

### DISCUSSION THREAD

**Discussion by subbotinvd** [night/betterttv#6020] [Ideas]
Admins, we aare waiting you :)

**Comment by Egalpra** [comment]
+1 We are really looking forward to when BTTV will support VkPlay Live (sorry for my google translate english)

**Comment by iamkito** [comment]
Hi! I will jump with happiness if the emoticons BTTV appear on VkPlay Live! Please do it =p pleeeeeeeeeeeeeeeeeeeeeeeeeeeeeeaase

**Comment by taotwi** [comment]
Please add BetterTTV support for VK Play that wil help all the streamers  a lot!

**Comment by SamVicLp** [comment]
Hi, im streamer from Vk Play Live, i need and we are need you, Bttv! Because you are very important part of streaming industry on the world... And after all sunctions, bad rules and toxic atmosphere on Twitch, big % of streamers relocate to Vk Play Live, and we want all smiles what we love - KEKW, GIGALUL, PEPECHILL AND MORE...
We need your support on Vk Play Live, its better CIS+ streaming platform, how Twitch was better on 2014-2017 years. And only one thing what this place can be comletely ideal - its you, BetterTV team!
With respect and love, streamer SamVicLp, leaved from Twitch, reborned on VK Play Live

**Comment by elphx** [comment]
We really need it!

**Comment by KOCCTON** [comment]
Good afternoon administration, we are waiting for your support VK PLAY LIVE

**Comment by re-baut** [comment]
Good night. I would be really gratefull if you started supporting of VL PLAY. This platform really need your help. Thanks in advnce.

**Comment by dude2d** [comment]
Please add BTTV for VK Play, i really enjoy bttv on twitch but now i miss it on vk play
